server/src/main/java/org/bonitasoft/console/common/server/page/CustomPageService.java

```diff
@@ -120,13 +121,13 @@ public void ensurePageFolderIsUpToDate(final APISession apiSession, final PageRe
     }
 
     @SuppressWarnings("unchecked")
-    public Class<PageController> registerPage(final GroovyClassLoader pageClassLoader, final PageResourceProvider pageResourceProvider)
+    public Class<PageController> registerPage(final GroovyClassLoader pageClassLoader, final PageResourceProviderImpl pageResourceProvider)
```

Comment:
Why are you asking for an Impl now? We shoudl still use the interface, no?
BTW, if the class ha sbeen updated, and the new interface doesn't inherits from the old one, we need to provide both methods and deprecates the old one